tools/state-viewer/src/cli.rs

```diff
@@ -74,30 +74,31 @@ impl StateViewerSubCommand {
         let near_config = load_config(home_dir, genesis_validation)
             .unwrap_or_else(|e| panic!("Error loading config: {:#}", e));
         let store_opener =
-            near_store::Store::opener(home_dir, &near_config.config.store).mode(mode);
+            near_store::NodeStorage::opener(home_dir, &near_config.config.store).mode(mode);
         let store = store_opener.open().unwrap();
+        let hot = store.get_store(near_store::Temperature::Hot);
         match self {
```

Comment:
This seems scary. Most of this functions will need a refactor to use NodeStorage to make them useful on archival nodes (I mean beyond rpc nodes). Doesn't need to happen right away, though. Probably.

Reply:
Maybe not.  At first we could provide --hot and --cold switches.